.github/workflows/build.yml

```diff
@@ -4,7 +4,10 @@ on: push
 
```

Comment:
[can't comment on unchanged lines]

Can you change `on: push` to `on: pull_request`? It's not running the GH Actions for your PR and I _think_ this is the culprit. Once this changes, I should theoretically get the ["Approve and run"](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks) button

Reply:
Just kidding, remembered I can do it myself :upside_down_face: